input/config/ubuntu/22.04/k8s.json

```diff
@@ -224,7 +204,7 @@
     },
     {
       "package": "registry.k8s.io/nfd/node-feature-discovery",
-      "tag": "v0.14.2",
+      "tag": "v0.12.1",
```

Comment:
Is there a reason for picking up and older version of NFD?

Reply:
nvidia device plugin  https://nvidia.github.io/k8s-device-plugin/stable/nvidia-device-plugin-0.14.4.tgz uses v0.12.1 , so we have reverted this image instead of latest.  
![image](https://github.com/user-attachments/assets/ec429b4a-bf8b-4d1f-b54f-3419f095b257)